src/apps/loan-list/materials/utils/digital-material-fetch-hoc.tsx

```diff
@@ -11,7 +11,8 @@ type InputProps = {
 
 const fetchDigitalMaterial =
   <P extends object>(
-    Component: ComponentType<P & MaterialProps>
+    Component: ComponentType<P & MaterialProps>,
+    FallbackComponent?: ComponentType
```

Comment:
This is really a minor. But I would use the word `LoadingComponent`because it is used when data is loading. `FallbackComponent` sounds to me like something you show if you failed loading data. 
Just a thought.